### DISCUSSION TITLE
Unable to send email from webmail or Android gmail app

### DISCUSSION THREAD

**Discussion by mjarvisal** [modoboa/modoboa#3123] [Help]
I don't know what is wrong and have no idea how to start debugging it but I cannot send email through the web interface or Android gmail app. I can see the mailbox in both. Don't know what broke it.

Only software that can send email is Mozilla Thunderbird in Windows 11.

**Comment by mjarvisal** [comment]
What logs should I be looking at to find the cause of this?
I have been looking at the /var/log/mail.log but there doesn't seem to be anything when I try to send a message from the webmail.
The webmail send button just goes gray and I cannot click it. No timeout or error is generated.

**Reply by tonioo** [reply]
Can you check the browser console? (press F12)

**Reply by mjarvisal** [reply]
I'm using Firefox and this is what the console gives as an error.:

Uncaught SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
    parseJSON jQuery
    sendmail webmail.js:1062
    jQuery 7
    sendmail webmail.js:1052
    jQuery 3

**Reply by mjarvisal** [reply]
Edge gives these errors:

Failed to load resource: the server responded with a status of 500 ()
webmail/#?action=com…cbcff912574a0203e:1  Uncaught SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
    at JSON.parse (<anonymous>)
    at Function.parseJSON (jquery.min.js:3:4322)
    at Webmail.<anonymous> (webmail.js:1062:26)
    at Object.i (jquery.min.js:3:6336)
    at c (jquery.min.js:3:7857)
    at Object.fireWith [as rejectWith] (jquery.min.js:3:8658)
    at k (jquery.min.js:5:14040)
    at XMLHttpRequest.r (jquery.min.js:5:18226)

**Reply by tonioo** [reply]
@mjarvisal Can you enable debug traces? (set DEBUG to True inside settings.py file and reload uwsgi)
